bioblend/galaxy/quotas/__init__.py

```diff
@@ -175,6 +181,11 @@ def update_quota(
                         non-default quota will throw an error. Passing None is
                         equivalent to not changing the current status.
 
+        :type default: str
+        :param default: Whether or not this is a default quota. Valid values
+                        are "no", "unregistered", "registered" and None. None is
+                        equivalent to "no".
```

Comment:
```suggestion
        :type quota_source_label: str
        :param quota_source_label: If set, quota source label to apply this quota operation to. Otherwise, the default quota is used.

```